sys/include/net/ng_ipv6/nc.h

```diff
@@ -123,6 +124,11 @@ typedef struct {
     uint8_t l2_addr_len;                    /**< Length of ng_ipv6_nc_t::l2_addr */
     uint8_t flags;                          /**< Flags as defined above */
```

Comment:
Why does this comment say ``as defined above``, when the only definition “above” is actually in RFC 4861? When I first read that comment, I expected there to be a typedef'd enum which contained all possible flags, and the type of flags to be ``ng_ipv6_nc_config_flags`` or something rather than ``uint8_t ``... Would that make sense or am I missing something here?
(After skimming through RFC 4861, I'm assuming ``flags`` refers to the M and O flags...)